website/docs/docusaurus.config.js.md

```diff
@@ -31,7 +27,7 @@ module.exports = {
 
 - Type: `string`
 
-Tagline is used in a number of places in your site including the title for the web page, sub headings, etc.
+The tagline for your website.
```

Comment:
Actually is this field strictly necessary? If we don't use it in the pages/index it's not used right?

Reply:
i think not really required, i think we can make it theme-specific. but It is used in theme-fallback layout now (which we can remove). Its only used as default title in @theme/Layout.

Reply:
but this PR is only about docs change. I'm gonna take a rest. do you want to remove it from required field of docusaurus.config and make it as themeConfig instead ?

I think we're still missing a doc page for @docusaurus-theme-classic Theme Configuration
It's still not clear what are the fields available. eg: navbar, footer, image